e2e/test/E2ETests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp2.1;net472;net451</TargetFrameworks>
```

Comment:
Question - .net core 2.1 is on [LTS support](https://dotnet.microsoft.com/download/dotnet-core), did it not receive a fix for this issue? Also, it looks like .net core 2.2 has already reached end-of-life. Would .net core 3.1 be a better option here?

Reply:
Per the work item filed against us, we can upgrade to 2.2.1, 2.1.7 or higher. Do you recommend doing 2.1.7 instead?

Reply:
I think .net core v2.1.16 seems like a better option, just because it is on LTS, and so should receive any security updates. .net core 2.2 is no longer supported.

Reply:
Also, does it work with .net core 3.1? It should still be able to target ns2.0 dll.

Reply:
Yes I just checked the compatibility chart. 3.1 should work. I'll make the change to target 3.1.